### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by ArcturusZhang**
# Cadl Ranch Contribution Checklist:

- [x] I have written a [scenario spec](../docs/writing-scenario-spec.md)
- [x] I have **meaningful** `@scenario` names. Someone can look at the list of scenarios and understand what I'm covering.
- [x] I have written a [mock API](../docs/writing-mock-apis.md)
- [x] I have used `@scenarioDoc`s for extra scenario description and to tell people **how to pass** my mock api check.

### CODE REVIEW

**Review by XiaofeiCao — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
I'd like to include another bug fix here https://github.com/Azure/cadl-ranch/pull/640

I will approve this bump after that merged.

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**
